src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const nodes = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+const frag = document.createDocumentFragment();
+
+const sortedList = Array.from(nodes).sort((a, b) =>
+  parseInt(a.dataset.salary.replace(/\D/g, ''))
+- parseInt(b.dataset.salary.replace(/\D/g, '')));
```

Comment:
It`s better to move this variable into sortList function, because it belongs to sortList

Reply:
Thanks, but have a next problem if I move this into sortListFunction:
The function "getEmployees()" use this const too, so for fixing your comment, I just move "sortedList" into two of these functions